src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -489,7 +489,7 @@ function showVersionWarningBanner(data) {
   middle.classList = "bd-header-announcement__content";
   inner.classList = "sidebar-message";
   button.classList =
-    "sd-btn sd-btn-danger sd-shadow-sm sd-text-wrap font-weight-bold ms-3 my-1 align-baseline";
```

Comment:
I carry over `btn` and `text-wrap` but not `btn-danger` and `shadow-sm`.

Bootstrap `btn-danger` applies colors that I have to override with `pst-` colors, so it's not useful. And `shadow-sm` isn't needed because we have the `box-shadow` mixin.